bootstrap.sh

```diff
@@ -360,7 +364,7 @@ if test "x$flag_no_python" = x; then
 import python ;
 if ! [ python.configured ]
 {
-    using python : $PYTHON_VERSION : $PYTHON_ROOT ;
+    using python : $PYTHON_VERSION : $PYTHON_ROOT : $PYTHON_INCLUDE ;
```

Comment:
Consider double-quoting `$PYTHON_INCLUDE` to allow it to work on Windows. (I'm also double-quoting `$PYTHON_ROOT` in #300.)